DashAI/front/src/components/experiments/ModelsTable.jsx

```diff
@@ -109,6 +109,9 @@ ModelsTable.propTypes = {
     name: PropTypes.string,
     dataset: PropTypes.object,
     task_name: PropTypes.string,
+    input_columns: PropTypes.arrayOf(number),
+    output_columns: PropTypes.arrayOf(number),
+    splits: PropTypes.object,
```

Comment:
Maybe you could define here that split is a json with train,validation and test.

```suggestion
    splits: PropTypes.shape({
        train: PropTypes.number,
        validation: PropTypes.number,
        test: PropTypes.number
  })
```